src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(first) {
+    return function(second) {
+      if (counter >= 3 && counter % 2 > 0) {
+        counter++;
+
+        return 'Bzzz... Error!';
+      } else {
+        counter++;
```

Comment:
just write counter++ outside of if block, you need to increase counter every time